aleph/metrics/flask.py

```diff
@@ -0,0 +1,93 @@
+from urllib.parse import urlparse
+from flask import request
+from werkzeug.exceptions import NotFound
+from timeit import default_timer
+from prometheus_client import (
+    generate_latest,
+    CollectorRegistry,
+    Counter,
+    Histogram,
+    CONTENT_TYPE_LATEST,
+)
+from prometheus_client.multiprocess import MultiProcessCollector
+from aleph.settings import SETTINGS
+
+METRICS_ENDPOINT_NAME = "metrics"
+
+REQUEST_DURATION = Histogram(
+    "http_request_duration_seconds",
+    "Duration of requests to the Aleph API in seconds",
+    ["method", "status", "api_endpoint"],
+)
+
+REQUEST = Counter(
+    "http_request_total",
+    "Total number of Aleph API requests",
+    ["method", "status", "api_endpoint", "logged_in"],
+)
```

Comment:
Should these be prefixed with `aleph_`, too?

Reply:
Added the prefix for these metrics too as it makes it much more convenient to filter out only metrics related to Aleph.